doc/Developing.md

```diff
@@ -53,7 +53,7 @@ For Windows instructions [click here](#building-on-windows).
 ## Prerequisites
 
  - Node.js `>= 12.14.1` **AND** `< 13`.
-   - Preferably, **use** Node version [`12.14.1`](https://nodejs.org/download/release/v12.14.1/),  as it is the the recommended minimum version according to the framework's supported `electron` version.
+   - Preferably, **use** Node version [`12.14.1`](https://nodejs.org/download/release/v12.14.1/),  as it is the recommended minimum version according to the framework's supported `electron` version.
```

Comment:
minor: since you are modifying the line already, do you mind removing the extra whitespace?

```suggestion
   - Preferably, **use** Node version [`12.14.1`](https://nodejs.org/download/release/v12.14.1/), as it is the recommended minimum version according to the framework's supported `electron` version.
```